src/PluginEntry/PluginEntry.scss

```
@@ -0,0 +1,4 @@
+.page-header {
```

```suggestion
.hacInfra-page-header {
```

so this class is namespaced?

---

Related:  wondering if we want to adopt [the same directory structure `hac-dev` is using](https://github.com/openshift/hac-dev/tree/main/src) and move these files to `src/pages/PluginEntry.tsx`?

---

Thanks @rhamilto. Regarding namespacing the class, do we follow a particular convention for the prefix (camel case vs kebab case)? Perhaps we can have the prefix in kebab case too?

As for the directory structure, I was following the original structure that was proposed by @andrewballantyne in the Readme [here](https://github.com/openshift/hac-infra/blob/63573cc1335ff07f42c8bfb5b1180e7a0421f1d3/README.md#folder-structure-design) where the folder name reflected the underlying external component and was in Pascal case. Are you recommending `pages` as a top level folder for all the component folders?

---

Just because you tagged me, I'll share my opinion 🙂 imo, you'll never get consistency in HAC. Roll your own, copy someones or whatever you want -- no matter what the structure is, it is unlikely to ever unify between known HAC plugins and definitely not any other plugin that wants to share in the HAC ecosystem.